packages/components/src/flex/flex/hook.ts

```diff
@@ -12,18 +12,18 @@ import deprecated from '@wordpress/deprecated';
 /**
  * Internal dependencies
  */
-import { useContextSystem } from '../../ui/context';
+import { useContextSystem, WordPressComponentProps } from '../../ui/context';
 import { useResponsiveValue } from '../../ui/utils/use-responsive-value';
 import { space } from '../../ui/utils/space';
 import * as styles from '../styles';
 import { useCx, rtl } from '../../utils';
+import type { FlexProps } from '../types';
+
+function useDeprecatedProps(
+	props: WordPressComponentProps< FlexProps, 'div' >
+): WordPressComponentProps< FlexProps, 'div' > {
+	const { isReversed, ...otherProps } = props;
```

Comment:
Curious to know if there's any particular reason for moving the props de-structuring from the function's signature to the body?

Reply:
No special reason. Just felt consistent with how the component and `useFlex` does it. No strong option here so can revert if you want.

Reply:
Don't worry, it's ok the way it is — no strong personal preference either!